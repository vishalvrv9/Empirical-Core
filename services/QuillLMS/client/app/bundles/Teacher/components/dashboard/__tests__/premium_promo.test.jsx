import React from 'react';
import { shallow } from 'enzyme';
import MockDate from 'mockdate';

import PremiumPromo from '../premium_promo';

describe('PremiumPromo component', () => {
  it('should render', () => {
    expect(shallow(<PremiumPromo />)).toMatchSnapshot();
  });
});
