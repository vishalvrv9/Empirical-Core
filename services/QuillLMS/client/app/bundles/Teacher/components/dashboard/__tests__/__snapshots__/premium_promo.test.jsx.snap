// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PremiumPromo component should render 1`] = `
<div
  className="mini_container results-overview-mini-container col-md-4 col-sm-5 text-center"
>
  <div
    className="mini_content premium-promo"
  >
    <div
      className="gray-underline"
      style={
        Object {
          "position": "relative",
        }
      }
    >
      <h3>
        Get Quill Premium Now
      </h3>
    </div>
    <p>
      Upgrade now and cover all of the students in your classes or school.
    </p>
    <div
      className="flex-row space-around"
    >
      <div
        className="pricing"
      >
        <h2>
          $80
        </h2>
        <span>
          per teacher
        </span>
      </div>
      <div
        className="fake-border"
      />
      <div
        className="pricing"
      >
        <h2>
          $900*
        </h2>
        <span>
          per school
        </span>
        <br />
        <span
          className="special-price"
        >
          *special price
        </span>
      </div>
    </div>
    <a
      className="q-button text-white"
      href="https://support.quill.org/quill-premium"
      target="_blank"
    >
      Learn More About Premium
    </a>
  </div>
</div>
`;
